dev/release/setup-rhel-rebuilds.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh -l
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+# A script to install dependencies required for release
+# verification Red Hat Enterprise Linux 8 clones in particular
+# on AlmaLinux 8 and Rocky Linux 8
+
+dnf -y update
+dnf clean all
+dnf -y install \
+  dnf-plugins-core \
+  yum-utils
+dnf config-manager --set-enabled powertools
+dnf -y update
+dnf -y module disable ruby
+dnf -y module enable ruby:2.7
+dnf -y groupinstall "Development Tools"
+dnf -y install \
+  epel-release \
+  ninja-build \
+  libcurl-devel \
+  python3-pip \
+  cmake \
+  git \
+  ncurses-devel \
+  gobject-introspection-devel \
+  libffi-devel \
+  openssl-devel \
+  maven \
+  java-1.8.0-openjdk-devel \
+  wget \
+  readline-devel \
+  gdbm-devel \
+  ruby-devel \
+  llvm-toolset \
+  llvm-devel
+dnf -y update
```

Comment:
Why is this needed?

Reply:
Use latest versions of dependencies https://dnf.readthedocs.io/en/latest/command_ref.html?highlight=upgrade#upgrade-command Can manage without these, though would take advantages of any improvements. Let me know if prefer to not have `dnf -y update`